src/ngx_postgres_util.c

```diff
@@ -45,8 +45,6 @@ void
 ngx_postgres_upstream_finalize_request(ngx_http_request_t *r,
     ngx_http_upstream_t *u, ngx_int_t rc)
 {
-    ngx_time_t  *tp;
```

Comment:
Instead of removing this line, we should just add an `#if` guard here as well. There is no point in defining this variable for the nginx 1.9.1+ code path anyway.